### PR TITLE
Issue3538 air2air recovery

### PR DESCRIPTION
**Pull request by SenHuang19**

### CODE REVIEW

**Comment by SenHuang19** [comment]
@JayHuLBL Can you please kindly review this PR?

File: Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Wheel.mo [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    "Type of controller";
+  parameter Real P_nominal(unit="W") = 1000
+    "Power at design condition";
+  parameter Real k(final min=0) = 1
+    "Gain of controller";
+  parameter Modelica.Units.SI.Efficiency epsS_cool_nominal(final max=1) = 0.8
+    "Nominal sensible heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsL_cool_nominal(final max=1) = 0.8
+    "Nominal latent heat exchanger effectiveness at the cooling mode";
+  parameter Modelica.Units.SI.Efficiency epsS_cool_partload(final max=1) = 0.75
```
Comment (JayHuLBL) [inline Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Wheel.mo:25]: should the `0.75` be a hard-coded the value? I see the value is used in the comment. If not, then we should not put the `75%` in the comment.

File: Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Wheel.mo [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+          fillPattern=FillPattern.Solid),
+        Rectangle(
+          extent={{32,-56},{94,-64}},
+          lineColor={0,0,255},
+          pattern=LinePattern.None,
+          fillColor={0,0,0},
+          fillPattern=FillPattern.Solid)}), Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    Documentation(info="<html>
+Model for a a generic, sensible and latent air-to-air heat exchanger, (heat recovery wheel), that consists of
```
Comment (JayHuLBL) [inline Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Wheel.mo:259]: typo `a a`

File: Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Validation/HeatExchagerWithInputEffectiveness.mo [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+<li>Temperature of the supply air, <i>TSup</i>, changes from <i>273.15 + 30 K</i> to <i>273.15 + 40 K</i> during the period from <i>120s</i> to <i>180s</i>;
+</ul>
+<ul>
+<li>Sensible heat exchanger effectiveness, <i>epsS</i>, changes from <i>0.7</i> to <i>0.8</i> during the period from <i>120s</i> to <i>180s</i>;
+</ul>
+<ul>
+<li>Latent heat exchanger effectiveness, <i>epsL</i>, changes from <i>0.7</i> to <i>0.8</i> during the period from <i>60s</i> to <i>120s</i>;
+</ul>
+
+<b>Note:</b> This problem may fail to translate in Dymola 2012 due to an error in Dymola's support
```
Comment (JayHuLBL) [inline Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/BaseClasses/Validation/HeatExchagerWithInputEffectiveness.mo:146]: add `<p> </p>`

File: Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+    epsL_heat_partload=0.6)
+    "Wheel"
+    annotation (Placement(transformation(extent={{6,-4},{26,16}})));
+    Modelica.Blocks.Sources.Ramp DamPos(
+    height=1.0,
+    duration=60,
+    offset=0,
+    startTime=60) "Damper position"
+    annotation (Placement(transformation(extent={{-80,-40},{-60,-20}})));
+  inner Modelica.Fluid.System system
```
Comment (JayHuLBL) [inline Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo:94]: is this instance needed? There is also one in `VarSpe.mo`.

File: Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+<p>
+The input signals are configured as follows:</p>
+<ul>
+<li>The temperature of the supply air, <i>TSup</i>, changes from <i>273.15 + 30 K</i> to <i>273.15 + 40 K</i> during the period from <i>60s</i> to <i>120s</i>.
+On the other hand, the temperature of the exhaust air is constant;
+</ul>
+<ul>
+<li>The bypass damper position, <i>DamPos</i>, changes from <i>0</i> to <i>1</i> during the period from <i>60s</i> to <i>120s</i>;
+</ul>
+<b>Note:</b> This problem may fails to translate in Dymola 2012 due to an error in Dymola's support
```
Comment (JayHuLBL) [inline Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo:145]: Is this statement needed?

**Review by JayHuLBL — COMMENTED** [review]
@SenHuang19 Would you please also add the release note into the `Buildings.package.mo`?

File: Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo (hunk 4 above)
Comment (SenHuang19) [reply Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/Examples/BypDam.mo:94]: @JayHuLBL just saw that you merged this PR, Thanks a lot. 
I was trying to get rid of this instance but got the following message when translating the model:

```
Note: 
Your model is using an outer "system" component but
an inner "system" component is not defined.
For simulation drag Modelica.Fluid.System into your model
to specify system properties.
```

Can you please let me know if there are any fixes to it?

**Pull request merged**